src/lib/y2partitioner/actions/controllers/lvm_vg.rb

```diff
@@ -77,6 +98,13 @@ def vg_size
           vg.size
         end
 
+        # Size of the logical volumes belonging to the current volume group
+        #
+        # @return [Y2Storage::DiskSize]
+        def lvs_size
+          vg.lvm_lvs.map(&:size).inject(Y2Storage::DiskSize.zero, :+)
```

Comment:
NP: just for your info, there is a `DiskSize.sum` method. In general it's slightly nicer, like
```ruby
Y2Storage::DiskSize.sum(vg.lvm_lvs.map(&:size))
```

Not a big deal, anyways.

Reply:
Nice to know :)